VideoPlayer/Android/README.md

```diff
@@ -0,0 +1,42 @@
+Zapp Plugin Player Example Android 
+==================
+
+The ZappPluginPlayerExample-Android is an example project for creating a player plugin for the Applicaster Zapp Platform. You can use this example project as a reference for how to build your own video player plugin.
+
+If you are not familiar with Zapp, please visit [Applicaster](http://www.applicaster.com) website for more details. The full [Zapp](http://zapp.applicaster.com) plugins documentation is available from within the Zapp system.
+
+### ZappPlayerAdapter
+* Implement PlayerContract with Android native VideoView and MediaController.
```

Comment:
Maybe something like:

This ZappPlayerAdapter is the main class connecting your plugin with the underlying core features of the SDK. As you can see in the implementation, the adapter is extending the BasePlayer class. This base player class is a helper class which does most of the heavy lifting for you and is implementing the PlayerContract directly (so you don't have to). The class will provide a native VideoView and a MediaController for you to use.  

The player has two different ways to be rendered; full-screen or inline. 

#### Full Screen Player
The full screen player will be displayed in it's own separate activity.

#### Inline Player
The inline player can be placed directly in any container view provided in order to render the player with in it. This means that no other activity will be created and the playback will take place directly within the view in which you placed the player.